bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -114,6 +114,12 @@ class App extends Component {
     let { userList } = this.props;
     const { compactUserList } = this.state;
 
+    //Variables for resizing user-list.
+    const USERLIST_MIX_WIDTH_PX = 100;
```

Comment:
Mix?

Reply:
Oh my God
I will fix it right now and re-push it sorry